tasks/kernels/run.py

```diff
@@ -16,33 +18,38 @@
 )
 from tasks.kernels.env import KERNELS_FAASM_USER
 
+MESSAGE_TYPE_FLUSH = 3
+NUM_PROCS = [2, 4, 8, 16]
+
 ITERATIONS = 1000
 SPARSE_GRID_SIZE_2LOG = 10
 SPARSE_GRID_SIZE = pow(2, SPARSE_GRID_SIZE_2LOG)
 
-NUM_PROCS = [1, 2, 3, 4, 5]
-
 PRK_CMDLINE = {
     "dgemm": "{} 500 32 1".format(
         ITERATIONS
-    ),  # iterations, matrix order, outer block size (?)
-    "nstream": "{} 2000000 0".format(
-        ITERATIONS
+    ),  # iterations, matrix order, outer block size
+    "nstream": "{} 20000 0".format(
+        1000000,
```

Comment:
I deliberately leave hardcoded numbers as arguments to format to make very clear that they are the number of iterations, and are subject to be changed.